contracts/test/veLON/vBalance.t.sol

```diff
@@ -54,6 +55,42 @@ contract TestVeLONDeposit is TestVeLON {
         assertEq(veLon.totalvBalance(), total);
     }
 
+    function testCreateLock_and_Withdraw_DifferentTime() public {
+        // alice and bob despoit for different amount and time
+        uint256 totalBalance = 0;
+        uint256 aliceStakeAmount = 1 * 365 days;
+        uint256 aliceTokenId = _stakeAndValidate(alice, aliceStakeAmount,  2 weeks);
+        uint256 initialAliceBalance = _initialvBalance(aliceStakeAmount, 2 weeks);
+        totalBalance = totalBalance.add(initialAliceBalance);
+        assertEq(veLon.vBalanceOf(aliceTokenId), initialAliceBalance);
+        assertEq(veLon.totalvBalance(), totalBalance);
+
+        uint256 bobStakeAmount = 5 * 365 days;
+        uint256 bobTokenId = _stakeAndValidate(bob, bobStakeAmount, 1 weeks);
+        uint256 initialBobBalance = _initialvBalance(bobStakeAmount, 1 weeks);
+        totalBalance = totalBalance.add(initialBobBalance);
+        assertEq(veLon.vBalanceOf(bobTokenId), initialBobBalance);
+        assertEq(veLon.totalvBalance(), totalBalance);
+
+        // check the totalSupply
+        assertEq(veLon.totalSupply(), 2);
+
+        // fast forward 1 weeks, bob's lock is expired
+        uint256 dt = 1 days;
+        vm.roll(block.timestamp + 7 * dt);
+        vm.warp(block.number + 1);
+        emit log(Strings.toString(veLon.vBalanceOf(bobTokenId)));
+        emit log(Strings.toString(veLon.vBalanceOf(aliceTokenId)));
+        emit log(Strings.toString(veLon.vBalanceOfAtTime(aliceTokenId, block.timestamp)));
+        // sub Bob's all balance
+        totalBalance = totalBalance.sub(initialBobBalance);
+        // sub Alice's delined balance ()
+        totalBalance = totalBalance.sub((1 * 7 days));
+        vm.prank(bob);
+        veLon.withdraw(bobTokenId);
+        assertEq(veLon.totalvBalance(), totalBalance, "expectedBalance is not equal");
```

Comment:
In the last assertion , the expected totalBalance is **604800**, but actual is **604799**